### DISCUSSION TITLE
Same timeline for the whole family (partner)

### DISCUSSION THREAD

**Discussion by DeltaTango69** [immich-app/immich#13093] [Q&A]
Hi,

our usecase is that we want all photos and videos from all the family (first only the partner) together in one timeline. The timeline should include also the same information of persons.

I have found the partner sharing . But when I share with my partner - there is no person information for the partner available?! Or I don't find the right button?

Also the partner sharing shares my whole set of photos and videos. Is there no way to share maybe only all photos and videos after for example 01.10.2008 ? Is that a good idea to share all photos and videos from my ex-partners ;-)

If these points are still missing, I would be happy to add them to the “Make a wish” list :-)

**Comment by bo0tzz** [comment]
This isn't possible yet. #12614

**Comment by DeltaTango69** [comment]
OK, thanks for the info - I'm excited to see what you come up with. :-) Shared libraries would be useful, for example - so far everything is tied to people. Why do all users have to save the same thumbs for the same external library? The concept is tied to people and should be tied to media and rights. So all media/thumbs/person data is only in one place and the user with admin rights then controls the sharing. In my opinion, this approach would be much more resource-efficient and also easier to keep track of.
